.travis.yml

```diff
@@ -33,11 +32,18 @@ env:
     - GOOGLE_APPLICATION_CREDENTIALS=${TRAVIS_BUILD_DIR}/credentials.json
     - CLOUDSDK_ACTIVE_CONFIG_NAME=getting-started-php
     - TEST_BUILD_DIR=${TRAVIS_BUILD_DIR}
+  matrix:
+    - STEP_NAME=1-hello-world
+    - STEP_NAME=2-structured-data
+    - STEP_NAME=3-cloud-storage
+    - STEP_NAME=4-auth
+    - STEP_NAME=5-logging
+    - STEP_NAME=6-compute-engine
 
```

Comment:
This will cause total of 18 matrices (3 php vers x 6 steps) which I'd like to avoid.

Can you keep the current PHP version based matrix and add the new subdirectories in the main phpunit.xml?